wrappers/node/demo/faber.js

```diff
@@ -182,6 +185,10 @@ async function runFaber (options) {
     {
       name: 'degree',
       restrictions: { 'attr::degree::value': 'maths' }
+    },
+    {
+      name: 'nickname',
```

Comment:
How is this attribute name linked to the attribute provided by Alice? An attribute named `nickname` is required by Faber, but Alice seems to provide an attribute named `attribute_3`. If there were multiple self attested attributes required and provided - how do they correspond in the resulting proof presentation?

Reply:
Touche, its done this way in demo for simplicity - real client would probably have to presentation request attachment, parse out mapping `<attribute referrent> (aka attribute_3)` -> `<claim_name> (aka nickname)`. I am added print statement 
```
logger.debug(`Proof request presentation attachment ${JSON.stringify(requestInfo, null, 2)}`)
```
on line 105. Ofcourse the whole think could done be better, the whole demo has grown into size where a simple libvcx based didcom nodejs client could be extracted and details like this could be hidden behind layer of abstraction. The demo code is too long and complex now, in fact I'd like to simplify the whole demo later in separate PR.